### PR TITLE
replacing long int and unsigned long int

### PR DESCRIPTION
**Pull request by tskille**
All EclIO classes and associated tests has been checked and updated. This includes cpp file for python bindings

  - long int -> int64_t
  - unsigned long int -> uint64_t

### CODE REVIEW

File: src/opm/io/eclipse/EclFile.cpp [hunk 1]
```diff
@@ -158,8 +158,8 @@ unsigned long int sizeOnDiskBinary(long int num, Opm::EclIO::eclArrType arrType)
             int maxBlockSize = std::get<1>(sizeData);
             int maxNumberOfElements = maxBlockSize / sizeOfElement;
 
-            long unsigned int numBlocks = static_cast<unsigned long int>(num)/static_cast<unsigned long int>(maxNumberOfElements);
-            long unsigned int rest = static_cast<unsigned long int>(num) - numBlocks*static_cast<unsigned long int>(maxNumberOfElements);
+            long unsigned int numBlocks = static_cast<uint64_t>(num)/static_cast<uint64_t>(maxNumberOfElements);
+            long unsigned int rest = static_cast<uint64_t>(num) - numBlocks*static_cast<uint64_t>(maxNumberOfElements);
```
Comment (joakim-hove) [inline src/opm/io/eclipse/EclFile.cpp]: Here I would use `auto numBlocks = `

**Comment by joakim-hove** [comment]
jenkins build this please

File: src/opm/io/eclipse/EclFile.cpp (hunk 1 above)
Comment (tskille) [reply src/opm/io/eclipse/EclFile.cpp]: Thanks, sorry that I missed a few there. Updated now.

**Comment by joakim-hove** [comment]
jenkins build this please

**Pull request merged**